### PR TITLE
feat(groups): add command to migrate tihlde members to tihlde group

### PR DESCRIPTION
**Pull request by Eirsteir**
## Proposed changes
Add management command to migrate all TIHLDE members to our TIHLDE group.

The command can be run with `docker-compose run --rm web python manage.py migrate_members`

Issue number: No issue

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] CHANGELOG.md has been updated. [Guide](https://tihlde.slab.com/posts/changelog-z8hybjom)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Pull request title follows [conventional commits](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) (`type(scope): description`)
- [x] Build (`make PR`) was run locally without failures

### CODE REVIEW

**Review by olros — APPROVED** [review]

**Review by Her0elt — APPROVED** [review]

**Review by Zenjjim — APPROVED** [review]

File: app/tests/groups/test_migrate_members_command.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+    call_command(MIGRATE_COMMAND)
+
+    tihlde_members = User.objects.all()
+
+    assert (
+        user.membership.filter(group__slug=GROUPS_TIHLDE_SLUG).exists()
+        for user in tihlde_members
+    )
+    assert Membership.objects.count() == batch_size
```
Comment (norgor) [inline app/tests/groups/test_migrate_members_command.py:34]: You are asserting more than the test implies. You are checking that users are migrated but also that no other memberships are created. Should this be it's own test?

**Review by norgor — COMMENTED** [review]
Looks very nice! One small thing about one of the tests. 🐍

File: app/tests/groups/test_migrate_members_command.py (hunk 1 above)
Comment (Eirsteir) [reply app/tests/groups/test_migrate_members_command.py:34]: Good catch! Thanks for pointing it out 😄

Comment (Eirsteir) [reply app/tests/groups/test_migrate_members_command.py:34]: Good catch! Thanks for pointing it out 😄

**Pull request merged**